spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -237,96 +342,9 @@ public Message postProcessMessageBeforeSend(Message message, Event event) {
 		@Override
 		public void run() {
 			try {
-				RabbitTemplate rabbitTemplate = AmqpAppender.this.rabbitTemplate;
-				rabbitTemplate.setConnectionFactory(AmqpAppender.this.manager.connectionFactory);
 				while (true) {
```

Comment:
? 

`shutDownNow()` will interrupt the thread so we'll get an `InterruptedException` on the next `take()` from the queue.